languages/rust/oso/src/host/from_polar.rs

```diff
@@ -213,3 +214,482 @@ where
         }
     }
 }
+impl<A, B, C> FromPolar for (A, B, C)
+where
+    A: FromPolar,
+    B: FromPolar,
+    C: FromPolar,
+{
+    fn from_polar(_term: &Term, _host: &mut Host) -> crate::Result<Self> {
+        Err(crate::OsoError::FromPolar)
```

Comment:
Why are these the same trait if the implementation is an error for every term?

Reply:
Yeah probably don't have to be, could do it like ToPolar and to ToPolarResults where one feeds into the other. Don't really wanna mess with it now though.

Reply:
On follow up list.